Add Immersive Movement

sean-kang opened the pull request:
I added metadata for Immersive Movement mod.
https://www.nexusmods.com/skyrimspecialedition/mods/15649?tab=description

Thanks.

sean-kang committed:
Add Immersive Movement

MacSplody commented on masterlist.yaml:
```diff
@@ -6769,6 +6769,47 @@ plugins:
       # version: 1.06a
       - crc: 0x20AA5984
         util: 'SSEEdit v3.2.2'
+  - name: 'Immersive Movement.esp'
```
Overall placement generally doesn't make a difference, but I'd prefer if this were placed somewhere more appropriate. If you're unsure, then just place new entries at the end of the masterlist.

sean-kang replied:
Moved to Gameplay section.

MacSplody commented on masterlist.yaml:
```diff
@@ -6769,6 +6769,47 @@ plugins:
       - crc: 0x20AA5984
         util: 'SSEEdit v3.2.2'
+  - name: 'Immersive Movement.esp'
+    group: *lowPriorityGroup
```
This group assignment isn't necessary and will cause sorting errors.

sean-kang replied:
Removed.

MacSplody commented on masterlist.yaml:
```diff
@@ -6769,6 +6769,47 @@ plugins:
+  - name: 'Immersive Movement - Mortal Enemies Patch.esp'
+    after:
+      - 'Immersive Movement.esp'
+      - 'Mortal Enemies.esp'
```
An after is fine for Mortal Enemies but all Mortal enemy variants should be added.

`Mortal Enemies.esp`
`Mortal Enemies - No RunWalk Changes.esp`
`Mortal Enemies - Rival Remix.esp`
`Mortal Enemies - Rival Remix - NoRunWalkChanges.esp`
`Mortal Enemies - 2.0 Test Version.esp`

sean-kang replied:
Done. Also added the variants to the condition.

MacSplody commented on masterlist.yaml:
```diff
@@ -6769,6 +6769,47 @@ plugins:
+        util: 'SSEEdit v4.0.2'
+  - name: 'Immersive Movement - Mortal Enemies Patch.esp'
+    after:
+      - 'Immersive Movement.esp'
```
`Req:` would be better for `Immersive Movement.esp`.

sean-kang replied:
Done.

MacSplody commented on masterlist.yaml:
```diff
@@ -6769,6 +6769,47 @@ plugins:
+  - name: 'Immersive Movement - Smilodon Patch.esp'
+    after:
+      - 'Immersive Movement.esp'
+      - 'Smilodon - Combat of Skyrim.esp'
```
You can use `Req:` for both of these as well.

sean-kang replied:
Done.

MacSplody commented on masterlist.yaml:
```diff
@@ -6769,6 +6769,47 @@ plugins:
+  - name: 'Immersive Movement - Wildcat Patch.esp'
+    after:
+      - 'Immersive Movement.esp'
+      - 'Wildcat - Combat of Skyrim.esp'
```
Same `Req:` for both of these as well.

sean-kang replied:
Done.

sean-kang committed:
Fix Immersive Movement metadata

- Relocate to Gamplay section.
- Check variants of Mortal Enemies.
- Use req key to specify plugin dependencies.

sean-kang commented:
Thanks for the comments. I just pushed a new commit to fix them.

MacSplody reviewed (approved):
Nice work, Thanks.

MacSplody commented:
@sean-kang Thanks, congrats on your first contribution.

MacSplody commented:
#497
